frontend/src/store/modules/shoots/index.js

```diff
@@ -366,14 +383,23 @@ const mutations = {
   HANDLE_EVENTS (state, { rootState, rootGetters, events }) {
     const onlyShootsWithIssues = get(state, 'shootListFilters.onlyShootsWithIssues', true)
     let setFilteredItemsRequired = false
+    const objectsToAdd = []
+    const shouldAddShoot = (shoot) => {
+      // Do not add healthy shoots when onlyShootsWithIssues=true, this can happen when toggeling flag
+      return rootState.namespace !== '_all' ||
+      !onlyShootsWithIssues ||
+      onlyShootsWithIssues === shootHasIssue(shoot)
+    }
     forEach(events, event => {
       switch (event.type) {
         case 'ADDED':
+          if (shouldAddShoot(event.object)) {
+            objectsToAdd.push(event.object)
+            setFilteredItemsRequired = true
+          }
+          break
         case 'MODIFIED':
-          if (rootState.namespace !== '_all' ||
-            !onlyShootsWithIssues ||
-            onlyShootsWithIssues === shootHasIssue(event.object)) {
-            // Do not add healthy shoots when onlyShootsWithIssues=true, this can happen when toggeling flag
+          if (shouldAddShoot(event.object)) {
```

Comment:
The old shoot had an issue but the new shoot does not have an issue anymore. We do not update the shoot in this case.